CHANGELOG.md

```diff
@@ -7,10 +7,11 @@ Release Versions:
 
 ## Upcoming changes (in development)
 
-- Transfer repository ownership to AICA (#1)
 - Add pkg-config files for control libraries (epfl-lasa#326)
 - Add pkg-config files for clproto (epfl-lasa#327)
 - Use pkg-config to check for dependencies (epfl-lasa#328)
+- Transfer repository ownership to AICA (#1)
+- Remove timestamp setter and rename getter (#2)
```

Comment:
Actually now the PR title and this line can be updated to be more specific, something like:
```suggestion
- Get the age of a state and remove timestamp setter (#2)
```